msg/JogJoint.msg

```diff
@@ -0,0 +1,11 @@
+ # Used in time-stamping the message.
+ Header header
+ 
+ # Name list of the joints. You don't need to specify all joints of the
+ # robot. Joint names are case-sensitive.
+ string[] joint_names
+ 
+ # Relative displacement of the joints to jog. The order must be
+ # identical to joint_names. Unit is radian for revolute joints,
+ # meter for linear joints.
```

Comment:
This assumes that the listed joints all operate in position mode. A mobile base wheel or other continuous joints are often able to handle both position and velocity requests. This is not always the case, please add a comment clarifying this.